### ISSUE TITLE
Random Word: Should show definition for the random word.

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/zeroclickinfo-spice#703]
The look of the random word instant answer is a bit plain:
![screen shot 2014-05-09 at 8 45 50 am](https://cloud.githubusercontent.com/assets/81969/2928122/e95a7f6c-d777-11e3-96eb-117ef7cfab13.png)

A better way of doing this is to get the random word from the endpoint and pass the resulting word to the dictionary IA which looks like this:
![screen shot 2014-05-09 at 8 48 07 am](https://cloud.githubusercontent.com/assets/81969/2928146/3401d52e-d778-11e3-891f-0205f95e5c8d.png)

In other words, the random word IA should load the dictionary IA. Loading the JS for the dictionary would work like this: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/maps/chains/maps_chains.js#L2

------ 

IA Page:  https://duck.co/ia/view/rand_word

**Comment by killerfish** [comment]
> in this case is there an easy way to preserve the RandomWord Spice's Metabar info

Anyway to achieve this guys?

**Comment by russellholt** [comment]
We don't have a general strategy for embedding other IAs yet.

**Comment by moollaza** [comment]
This has since been implemented.
